### ISSUE TITLE
Captchavel does not work in GET routes

### ISSUE THREAD

**Issue by Dimonka2** [DarkGhostHunter/Captchavel#12]
Hi, I wanted to add your recapcha middleware to Auth routes and of cause got the error message above. Does it make sense to have it? It would make much more sense just to ignore GET requests and let people to add middleware like:
```
Route::group(['middleware' => 'recaptcha'], function() {
    Auth::routes();
});
```
I am pretty new in Laravel and I have no idea how to find a workaround for this case..

**Comment by DarkGhostHunter** [comment]
While is working as intended, I think you are right. If you put the middleware in a group of routes that have a bunch of GET and POST routes, you will get the exception in the GET routes, so you will have to add the middleware to each POST route manually.

Technicality, the reCAPTCHA verification can also work on GET requests, by seeking the g-recaptcha-response in the query parameters on the URI, but seems very edge case. In that case, it's better to use authenticatio or authorization.

Meanwhile, you can override this middleware by creating your own, extending the Captchavel middleware, and registering yours in your HTTP Kernel.

**Comment by Dimonka2** [comment]
I know that you put in documentation that the GET returns an exception. For me the actual use of recaptcha was to protect only authorization part of my project. 

Can you give me some tip how to extend the Captchavel middleware? As I mentioned before I am pretty new to Laravel and any hint would be very appreciated.
Thank you!

**Comment by DarkGhostHunter** [comment]
Don't, I'm issuing an update on that for the next minutes.

**Comment by DarkGhostHunter** [comment]
[Fixed in v2.0.1](https://github.com/DarkGhostHunter/Captchavel/tree/v2.0.1).

Have a nice day.
